local_modules/mapbox_style_configure/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function configure(style, mapStyleConfig, lang) {
+  return JSON.parse(style
+    .replace(/\{locale\}/g, lang)
+    .replace('"{tileserver_base}"', mapStyleConfig.baseMapUrl)
+    .replace('"{tileserver_poi}"', mapStyleConfig.poiMapUrl)
+    .replace('{spriteserver}', mapStyleConfig.spritesUrl)
+    .replace('{fontserver}', mapStyleConfig.fontsUrl)
```

Comment:
`spriteserver` and `fontserver` need to be absolute urls.  
We could probably build it using `req` properties (similar to  "src/adapters/scene_config.js" )

Reply:
i think 
{ spritesUrl : `${req.get('host')}${config.mapStyle.spritesUrl}` ,
 fontsUrl : `${req.get('host')}${config.mapStyle.fontsUrl}`
}
is enough